docs/guide/getting-started.md

```diff
@@ -189,3 +189,32 @@ If you would like to use it, install it with `npm i -g nodemon`. Then you can ru
 ### Now you are ready to start developing!🎉 🎉 🎉 🎉 
 
 Next, feel free to read through other parts of the documentation and codebase that interest you, and assign yourself to an issue that you want to work on!
+
+## Creating a Pull Request
+
+To contribute your changes to the documentation and the codebase, you would need to create a Pull Request.
+
+A Pull Request refers to an object within GitHub that developers use to add incremental changes to a repository. A Pull Request consists of several commits that developers make in the respective branches of their own forks before they request for these changes to be added to one of the branches of the original repository.
+
+Let's take the example of an issue that one of the developers, [Aadit](https://github.com/aaditkamat), has already worked upon previously:
+
+![An example of a GitHub issue](/imgs/exampleissue.png)
+
+The documentation for this project is hosted through [Vitepress](https://vitepress.dev/) pulling content from [Alex's repo](https://github.com/AlexanderWangY/OSC-Proximity-Documentation). So to work on this issue, you will have to create a fork of Alex's repo and make changes locally before you create a PR. 
```

Comment:
Nah im just kidding, could you please remove all names from this documentation? You can link repos but please keep contacts/names within our contact.md